source/jormungandr/jormungandr/realtime_schedule/sytral.py

```diff
@@ -115,17 +115,19 @@ def _make_url(self, route_point):
         The url returns something like a departure on a stop point
         """
 
-        stop_id = route_point.fetch_stop_id(self.object_id_tag)
+        stop_id_list = route_point.fetch_all_stop_id(self.object_id_tag)
 
-        if not stop_id:
+        if not stop_id_list:
             logging.getLogger(__name__).debug(
-                'missing realtime id for {obj}: stop code={s}'.format(obj=route_point, s=stop_id),
+                'missing realtime id for {obj}: stop code={s}'.format(obj=route_point, s=stop_id_list),
                 extra={'rt_system_id': unicode(self.rt_system_id)},
             )
             self.record_internal_failure('missing id')
             return None
 
-        url = "{base_url}?stop_id={stop_id}".format(base_url=self.service_url, stop_id=stop_id)
+        url = self.service_url
+        for stop_id in stop_id_list:
+            url += "?stop_id={stop_id}".format(stop_id=stop_id)
```

Comment:
Does this work: http://bob.com/?stop_id=42?stop_id=43?
Shouldn't it be : http://bob.com/?stop_id=42&stop_id=43?

Reply:
To be honest, I don't know the format :smile_cat:

Reply:
Done